netbox/ipam/api/views.py

```diff
@@ -146,7 +148,8 @@ class FHRPGroupAssignmentViewSet(NetBoxModelViewSet):
 
 class VLANGroupViewSet(NetBoxModelViewSet):
     queryset = VLANGroup.objects.annotate(
-        vlan_count=count_related(VLAN, 'group')
+        vlan_count=count_related(VLAN, 'group'),
+        utilization=Round(F('vlan_count') / (F('max_vid') - F('min_vid') + 1.0) * 100, 2)
```

Comment:
The annotation logic should be moved to a queryset method to avoid duplication. Also, `Round()` doesn't seem to be applied consistently.